po/it.po

```diff
@@ -1872,68 +1863,68 @@ msgstr ""
 
 #: rust/libnewsboat/src/configpaths.rs:146
 msgid "Migrating configs and data from Newsbeuter's XDG dirs..."
-msgstr ""
+msgstr "Migrando configurazioni e dati da cartelle XDG di Newsbeuter..."
 
 #: rust/libnewsboat/src/configpaths.rs:202
 msgid "Migrating configs and data from ~/.newsbeuter/..."
-msgstr ""
+msgstr "Migrando configurazioni e dati da ~/.newsbeuter/..."
 
 #: rust/libnewsboat/src/configpaths.rs:214
-#, fuzzy
 msgid "Aborting migration because mkdir on `%s' failed: %s"
-msgstr "Errore: apertura del file cache `%s' fallita: %s"
+msgstr "Migrazione interrotta per fallimento di mkdir su `%s': %s"
 
 #: rust/libnewsboat/src/filterparser.rs:63
-#, fuzzy
 msgid "attribute name"
-msgstr "attributo non trovato"
+msgstr "nome attributo"
 
 #: rust/libnewsboat/src/filterparser.rs:64
+#, fuzzy
 msgid "one of: =~, ==, =, !~, !=, <=, >=, <, >, between, #, !#"
-msgstr ""
+msgstr "uno tra: =~, ==, =, !~, !=, <=, >=, <, >, between, #, !#"
 
 #: rust/libnewsboat/src/filterparser.rs:65
+#, fuzzy
 msgid "one of: quoted string, range, number"
-msgstr ""
+msgstr "uno tra: quoted string, range, number"
```

Comment:
Unlike `between`, these strings are just words, and can be translated. Again, I promise to fix this before the next translation period :)

Reply:
No problem, got it, I'll translate them